webpack.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    entry: ['babel-polyfill', './parser.js'],
+    output: {
+        path: './bin',
```

Comment:
Not sure that "./bin/" feels right.
I thought that was typically used for CLI scripts in Node.
What about "./client"?